wampproto/acceptor.py

```diff
@@ -80,7 +80,11 @@ def receive_message(self, msg: messages.Message) -> messages.Message:
             match method:
                 case Acceptor.ANONYMOUS:
                     request = auth.AnonymousRequest(msg.realm, msg.authid, msg.authextra)
-                    response = self._authenticator.authenticate(request)
+                    try:
+                        response = self._authenticator.authenticate(request)
+                    except Exception as e:
+                        return messages.Abort(messages.AbortFields({}, uris.AUTHENTICATION_FAILED, args=[list(e.args)]))
```

Comment:
you are putting a list inside a list, thats wrong
```suggestion
                        return messages.Abort(messages.AbortFields({}, uris.AUTHENTICATION_FAILED, args=list(e.args)))
```